### ISSUE TITLE
Team Dorothy Thomas Code Review

### ISSUE THREAD

**Issue by jgte29** [yscokgor/Greville#1]
Hello Team Greville,

I hope that you all are doing well. I would like to begin by commending you for all of the great work that you have done on the project. It is evident that you all put a significant amount of time and effort into your code. Below, I have included our team's evaluation of each of your files of code. If you would like additional feedback on your code, or would like help making corrections, please do not hesitate to reach out during class on Tuesday.

1. README.md
- This file is very helpful for navigating your team's repository, especially given the fact that some of the notebooks do not have the most descriptive names.
- One thing we recommend would be for your team to go back and slightly adjust the register of your file to be a little more professional. For example, ensuring proper spelling and capitalization and avoiding shorthands such as "r" for the word "are" would make this file more suited for an academic setting.

2.  Greville_Group_Code.ipynb
- It is a general convention to `pip` install and import all of your libraries and packages that you plan to use in your notebook at the top of the notebook for organization purposes.
- The title is not well suited for the file. We suggest changing the name to something like population_tables.ipynb
- Errors presented when running `wbdata.get_sources()` and `wbdata.get_indicators()`. Should correct these functions to be  `wbdata.get_source()` and `wbdata.get_indicator()` respectively.
- Additionally, there is an error when running `wbdata.get_countries()`. Change this to `wbdata.get_country()`
3. Untitled2.ipynb
- It is a general convention to `pip` install and import all of your libraries and packages that you plan to use in your notebook at the top of the notebook for organization purposes.
- The title is not well suited for the file. We suggest changing the name to something like agriculture_economy.ipynb
- There is an error when running `wbdata.get_topics()`. Change this to `wbdata.get_topic()`
- More errors presented when running `wbdata.get_sources()` and `wbdata.get_indicators()`. Should correct these functions to be  `wbdata.get_source()` and `wbdata.get_indicator()` respectively.
- For cleanliness, we recommend removing terminal commands like "ls" and "cd" from the notebook. Either carry out these commands in the terminal or delete these cells when you are finished with them.
4.  proj1_code.ipynb
- Comment out `pip` install functions once you have used them as it makes the notebook take up less memory through not needing to print out all the installation outputs, and looks cleaner.
- Good practice to not put comments on the same line as a `pip` installation (e.g. %pip install cufflinks # IF NECESSARY).
- Error presented when running `wbdata.get_indicators()`. Should correct this function to `wbdata.get_indicator()`.
5. Botswana.ipynb
- Comment out `pip` install functions once you have used them as it makes the notebook take up less memory through not needing to print out all the installation outputs, and looks cleaner.
- Good practice to not put comments on the same line as a `pip` installation (e.g. %pip install cufflinks # IF NECESSARY).
- Other than this, the notebook looks great!
6.  ag_production.ipynb
- In future versions of this file, it might be wise not to directly copy cells from the course's lecture notebooks. Even if the information from lecture is helpful, your team should probably try to change it a little to make it more original. Additionally, there may be pieces of the lecture notebook that are not helpful for your project.

In conclusion, with the exception of some formatting issues and usage of depreciated functions, which unfortunately just became invalid a few days ago so not your fault, your team did a good job on the coding section of the project. Best of luck with the final presentation.

**Comment by yscokgor** [comment]
Thanks for such a detailed code review! it is appreciated!
